i18n/sk.toml

```diff
@@ -0,0 +1,24 @@
+[category]
+other = "oblasť"
```

Comment:
There is lot's of options but to stay as close to the english language perhaps following would be better:
```suggestion
other = "kategória"
```

🤷‍♂